docs/website/docs/getting-started.md

```diff
@@ -71,14 +71,19 @@ Load package 1692364844.460054 is LOADED and contains no failed jobs
 ```
 
 `dlt` just created a database schema called **mydata** (the `dataset_name`) with a table **users** in it.
-[Take a look at it using built-in Streamlit app](reference/command-line-interface#show-tables-and-data-in-the-destination):
+
+### See the data and schema
```

Comment:
we often use the title "Explore the data"

Reply:
That looks and sounds better.

Reply:
Done.